pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -487,16 +487,37 @@ protected void setupDB(@NonNull Config config) {
 
         int maxConnections = config.getInt("portal.maxConnections");
         String dbUrl = config.getString("portal.dbUrl");
+        String dssDBUrl = config.getString("dss.dbUrl");
+        int dssMaxConnections = config.getInt("portal.maxConnections");//todo pegah add to vault
```

Comment:
why add to vault? maybe separate maxConnections for dss? dss.maxConnections

Reply:
sorry, I meant secret manager  instead of vault.